### PR TITLE
Replaced the use of `pep517` with `pyproject-hooks` because of the rename

### PR DESCRIPTION
**Pull request by KOLANICH**

### CODE REVIEW

**Comment by KOLANICH** [comment]
https://github.com/pypa/pyproject-hooks/pull/151

File: pyproject.toml
Comment (layday) [inline pyproject.toml:8]: Please use the same description as the repo's.

File: README.md
Comment (layday) [inline README.md:12]: As below.

File: tests/constraints.txt
Comment (layday) [inline tests/constraints.txt:7]: This should be equal to the minimum version specified in `pyproject.toml`. Did you experience an issue with 1.0.0?

**Comment by KOLANICH** [comment]
>This should be equal to the minimum version specified in pyproject.toml. Did you experience an issue with 1.0.0?

I personally don't (I don't install packages from pypi and in my pip installation the resolver is disabled), `1.0.0` works fine for me (though on my machine 2.0.1 is installed, spinned up a venv to test), but
https://github.com/pypa/pyproject-hooks/blob/main/pyproject.toml#L13 requires `1.1.0` and this makes `pip`s resolver angry.

**Comment by layday** [comment]
Ok, bump it in `pyproject.toml` too then please.

**Review by henryiii — APPROVED** [review]
Looks fine to me. We might not want to rush a release out once this is merged, as it will take time to make this rename available in other ecosystems. Conda-forge, for example, does not have the renamed package yet.

**Review by gaborbernat — APPROVED** [review]

**Comment by pradyunsg** [comment]
> We might not want to release for a while after this is merged, as it will take time to make this rename available in other ecosystems. Conda-forge, for example, does not have the renamed package yet.

Don't those redistributors *also* delay their ingestion of `build`, if they've got a delayed ingestion of other packages from PyPI?

**Comment by webknjaz** [comment]
> Conda-forge, for example, does not have the renamed package yet.

Will it not work for them to just use an older version, then? Pretty sure this is how all downstream packagers would operate — they'll just package things in the order needed for the depresolver. It's kinda pointless to cater for a case that wouldn't even happen.

**Comment by rossburton** [comment]
Speaking with my distro maintainer hat on, don't wait.  I'm waiting for a new release of build that uses flit and pyproject-hooks. :)

**Comment by henryiii** [comment]
@layday, would you be okay to rebase #537 once? (I can do it for you next week). It might be nice to get a release out soon with flit and this. I’m on vacation the rest of the week, so I’m not volunteering quite yet. Also can’t review that PR till next week either. :(

**Comment by layday** [comment]
Yes, please go ahead. Thanks for asking.

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------

**Comment by henryiii** [comment]
Looks okay to me. I say we merge unless anyone has objections.

**Comment by nanonyme** [comment]
Is this pending further review and approval or is it just waiting for someone to click the merge button?

**Comment by henryiii** [comment]
The latter.

**Pull request merged**
